mne_icalabel/features/tests/test_psd.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+from mne.datasets import testing
+from mne.io import read_raw
+from mne.preprocessing import ICA
+
+from mne_icalabel.features import get_psds
+
+directory = testing.data_path() / "MEG" / "sample"
+raw = read_raw(directory / "sample_audvis_trunc_raw.fif", preload=False)
+raw.pick_types(eeg=True)
+raw.load_data()
+ica = ICA(n_components=7, method="picard")
```

Comment:
Is there a reason you asked for 7 components? Else let's lower it to 5 as it will slightly speed up the fit and the test.

Reply:
No, I was testing something. We can set it to 5.